operator/pkg/operator/manifests/banyandb/templates/service.yaml

```diff
@@ -0,0 +1,60 @@
+# Licensed to Apache Software Foundation (ASF) under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Apache Software Foundation (ASF) licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+{{- $svc := .Spec.Service.Template }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Name }}-banyandb
+  namespace: {{ .Namespace }}
+  labels:
+    app: banyandb
+    operator.skywalking.apache.org/banyandb-name: {{ .Name }}
+    operator.skywalking.apache.org/application: banyandb
+    operator.skywalking.apache.org/component: service
+spec:
+  type: {{ $svc.Type }}
+  selector:
+    app: banyandb
+    operator.skywalking.apache.org/banyandb-name: {{ .Name }}
+  ports:
+    - port: 17912
+      name: grpc
+    - port: 2121
+      name: observability
+    - port: 6060
+      name: pprof
```

Comment:
Please remove these ports. We don't tend to expose them through a service.

Reply:
done